include/NAS2D/Renderer/Renderer.h

```diff
@@ -136,6 +136,8 @@ class Renderer
 
 	virtual void setViewport(const Rectangle<int>& viewport) = 0;
 	virtual void setOrthoProjection(const Rectangle<float>& orthoBounds) = 0;
+	void setResolution(const Point<float> newResolution);
```

Comment:
Resolution is a size quantity. It may make more sense as a `Vector`.

The use of `float` also stands out, though that is pre-existing practice, so probably best to leave as is.